app/controllers/fhir/r4/api_controller.rb

```diff
@@ -770,9 +785,43 @@ def status_not_found
     end
   end
 
-  def status_unprocessable_entity(errors = [])
+  # 422 response with specific validation errors.
+  def status_unprocessable_entity(resource, fhir_map, req_json)
+    outcome = FHIR::OperationOutcome.new(issue: [])
+
+    resource&.errors&.messages&.each do |attribute, errors|
+      next unless VALIDATION.key?(attribute) || attribute == :base
+
+      fhir_path = fhir_map&.dig(attribute, :path)
+
+      # Extract the original value from the request body using FHIRPath
+      if fhir_path&.present?
+        begin
+          value = nil
+          Rails.logger.silence do
```

Comment:
Why is a silence necessary here?

Reply:
There's just a ton of debug output that we don't care about. I'll add a comment.

Reply:
Added https://github.com/SaraAlert/SaraAlert/pull/743/commits/11d7f9ca86d6fa3416ce22bcba5ae88784f5575a